contracts/prize-pool/PrizePool.sol

```diff
@@ -719,11 +719,10 @@ abstract contract PrizePool is PrizePoolInterface, OwnableUpgradeable, Reentranc
     )
   {
     (uint256 exitFee, uint256 _burnedCredit) = _calculateEarlyExitFeeLessBurnedCredit(from, controlledToken, amount);
-    uint256 duration = _estimateCreditAccrualTime(controlledToken, amount, exitFee);
-    if (duration > maxTimelockDuration) {
-      duration = maxTimelockDuration;
+    uint256 durationSeconds = _estimateCreditAccrualTime(controlledToken, amount, exitFee);
+    if (durationSeconds > maxTimelockDuration) {
+      durationSeconds = maxTimelockDuration;
     }
-    return (duration, _burnedCredit);
```

Comment:
We should `return (durationSeconds, _burnedCredit);` no? 
Cause I'm not sure `_burnedCredit` will be returned automatically since there is a `_`.